src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -150,6 +152,7 @@ private function storeEntityToPublish($entity, string $property): void
             $this->deletedEntities[(object) [
                 'id' => $this->iriConverter->getIriFromItem($entity),
                 'iri' => $this->iriConverter->getIriFromItem($entity, UrlGeneratorInterface::ABS_URL),
+                'type' => $metadata->getShortName(),
```

Comment:
The IRI must be used if defined: https://github.com/api-platform/core/blob/master/src/JsonLd/Serializer/ItemNormalizer.php#L89 (could use the coalesce operator now, by the way).